charts/prometheus-pingdom-exporter/README.md

```diff
@@ -0,0 +1,64 @@
+# Prometheus Pingdom Exporter
+
+- **Source:** https://github.com/giantswarm/prometheus-pingdom-exporter
+
+[prometheus-pingdom-exporter](https://github.com/giantswarm/prometheus-pingdom-exporter) the prometheus-pingdom-exporter cares about preprocessing the pingdom uptime check results for consumption of by prometheus.
+
+## Introduction
+
+This chart creates a kubernetes deployment on a Kubernetes cluster using the Helm package manager.
+
+## Installing the Chart
+
+Install from remote URL with the release name `prometheus-pingdom-exporter` into namespace `monitoring`:
```

Comment:
imho we should not use a namespace here

Reply:
We did this already for the kubernetes-policy-chart... https://github.com/kiwigrid/helm-charts/tree/master/charts/kubernetes-policy-controller#installing-the-chart

Reply:
For opa the namespace is important because the service does a configmap lookup for policies so the default namespace would not  a good place that's why I added the namespace

Reply:
Could you please add this info to the readme too? Thanks!